packaging/installer/install-required-packages.sh

```diff
@@ -1202,6 +1206,61 @@ install_apt_get() {
 # -----------------------------------------------------------------------------
 # centos / rhel
 
+prompt() {
+  if [ "${NON_INTERACTIVE}" -eq 1 ]; then
+    echo >&2 "Running in non-interactive mode, assuming yes (y)"
+    echo >&2 " > Would have promptedfor ${1} ..."
```

Comment:
```suggestion
    echo >&2 " > Would have prompted for ${1} ..."
```

Reply:
Hmm..for some reason I don't see this applied. The PR diff says that it's not applied